pkg/webhook/validate_service.go

```diff
@@ -46,10 +47,20 @@ const (
 func ValidateRevisionTemplate(ctx context.Context, uns *unstructured.Unstructured) error {
 	content := uns.UnstructuredContent()
 
+	var mode DryRunMode
+	features := config.FromContextOrDefaults(ctx).Features
+	switch features.PodSpecDryRun {
+	case config.Enabled:
+		mode = DryRunEnabled
+	case config.Disabled:
+		return nil
```

Comment:
Is this right though? This basically skips the whole validation?
I'd expect disable means we don't use dry run to validate the template.

Reply:
At the moment dryrun is the only validation in this codepath, but we'd want to rearrange this if we had other validations here